cmd/single-ocsp/test/gentest.go

```diff
@@ -52,16 +52,18 @@ func toPEM(cert *x509.Certificate) []byte {
 func main() {
 	// Instantiate the PKCS11 key
 	var pkcs11 struct {
-		Module string
-		Token  string
-		PIN    string
-		Label  string
+		Module          string
+		TokenLabel      string
+		PIN             string
+		PrivateKeyLabel string
+		SlotID          float64
```

Comment:
Because it's a struct for a json config and encoding/json (per spec) uses doubles (float64's) to represent numbers in json.